### PR TITLE
Countdown video background added

### LINKED ISSUE #50: Countdown Video Background
- [ ] Add Video Background to Countdown

### PR DESCRIPTION
**Pull request by Adem68**
## POC

https://user-images.githubusercontent.com/21019611/107697077-d2715f80-6cc3-11eb-9200-f7d1cb4787bb.mp4

Fixes #50

### CODE REVIEW

File: lib/pages/home_page.dart [hunk 1]
```diff
@@ -21,14 +20,32 @@ class HomePage extends StatefulWidget {
...
 
-  void initPageSectionList() {
+  void _initializeScrollController() {
+    scrollController.addListener(() {
+      if (scrollController.offset == 0.0 &&
+          !scrollController.position.outOfRange) {
+        if (!mounted) return;
+        setState(() => isAppbarCollapsing = false);
+      }
+      if (scrollController.offset >= 9.0 &&
```
Comment (salihgueler) [inline lib/pages/home_page.dart:40]: Why 9 :D Why not 10/12 etc.. :D

Comment (salihgueler) [reply lib/pages/home_page.dart:40]: Please move this 9.0 to a constant value with a meaningful name

File: lib/core/components/drawer/drawer_list_tile.dart [hunk 2]
```diff
@@ -18,8 +18,16 @@ class DrawerListTile extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ListTile(
-      title: Text('$title'),
-      leading: Icon(icon ?? Icons.warning),
+      title: Text(
+        '$title',
+        style: TextStyle(
```
Comment (bgoktugozdemir) [inline lib/core/components/drawer/drawer_list_tile.dart:23]: Do not forget to use const.

File: lib/core/styles/theme_helper.dart [hunk 3]
```diff
@@ -17,6 +17,7 @@ class ThemeHelper {
     visualDensity: VisualDensity.adaptivePlatformDensity,
     accentColor: lightColor,
     primaryColor: darkColor,
+    canvasColor: ThemeHelper.darkColor,
```
Comment (bgoktugozdemir) [inline lib/core/styles/theme_helper.dart:20]: Why do we need this color if it is the same as the darkColor?

File: lib/pages/home_page.dart [hunk 4]
```diff
@@ -9,6 +8,8 @@ import '../widgets/footer_view.dart';
 import '../widgets/section_title.dart';
 import '../widgets/speakers_section.dart';
 
+const _scrollofset = 12.0;
```
Comment (bgoktugozdemir) [inline lib/pages/home_page.dart:11]:
```suggestion
const _scrolloffset = 12.0;
```

File: lib/widgets/countdown_section/countdown_section.dart [hunk 5]
```diff
@@ -1,30 +1,97 @@
...
 import 'package:festival_flutterturkiye_org/widgets/countdown_section/countdown_exports.dart';
+import 'package:video_player/video_player.dart';
 
 const double _paddingSmall = 24.0;
 const double _paddingMedium = 48.0;
 const double _paddingLarge = 72.0;
 
-class CountdownSection extends StatelessWidget {
-  const CountdownSection();
+class CountdownSection extends StatefulWidget {
```
Comment (bgoktugozdemir) [inline lib/widgets/countdown_section/countdown_section.dart:13]: Do we have to use StatefulWidget in the main class?

If it builds again, the whole child widgets will too. And there are some calculations in the child-classes.

**Review by bgoktugozdemir — CHANGES_REQUESTED** [review]

File: lib/core/styles/theme_helper.dart (hunk 3 above)
Comment (Adem68) [reply lib/core/styles/theme_helper.dart:20]: We need to canvasColor for changing drawer background color.

image

**Review by bgoktugozdemir — APPROVED** [review]
LGTM

**Pull request merged**